### PR TITLE
Draft datasource config integration tests

### PR DESCRIPTION
**Pull request by anthonyburdi**
Integration tests to make sure the DraftDatasourceConfigAction runs against a live app instance + GX Core. This is a prerequisite to releasing the GX Agent.

Note: Uses draft datasource configs in seed data added via https://github.com/greatexpectationslabs/mercury/pull/3166

### CODE REVIEW

File: tests/integration/actions/test_draft_datasource_config_action.py
Comment (Kilo59) [inline tests/integration/actions/test_draft_datasource_config_action.py:1]: Can we add an integration marker, CI step + codecov flag for these tests that rely on a running GX Cloud container?

Comment (anthonyburdi) [reply tests/integration/actions/test_draft_datasource_config_action.py:1]: Ooh good call. Yes we absolutely can.

Comment (anthonyburdi) [reply tests/integration/actions/test_draft_datasource_config_action.py:1]: Planning to do that in a separate or separate PRs.

Comment (anthonyburdi) [reply tests/integration/actions/test_draft_datasource_config_action.py:1]: WIP PR: https://github.com/great-expectations/cloud/pull/241

**Pull request merged**